app/graphql/types/error_message.rb

```diff
@@ -0,0 +1,9 @@
+module Types
+  class ErrorMessage < BaseObject
+    field :errors, String, null: false
+
+    def errors
+      object.errors.full_messages
+    end
+  end
+end
```

Comment:
This little sign at the end of the file means that you do not have this VS Code setting turned on:

![image](https://user-images.githubusercontent.com/2687/57205212-5206d080-7000-11e9-8c60-5f84a711a854.png)

Rubocop (and maybe ESLint too?) will complain if you do not have this final newline at the end of the file. Check that setting in VS Code to prevent these issues.